### PR TITLE
add language identifier

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/framework/winforms/advanced/how-to-write-user-settings-at-run-time-with-csharp.md [hunk 1]
```diff
@@ -8,27 +8,27 @@ helpviewer_keywords:
...
-1.  Access the setting and assign it a new value as shown in this example:  
+1. Access the setting and assign it a new value as shown in this example:  
   
-    ```  
-    // C#  
-    Properties.Settings.Default.myColor = Color.AliceBlue;  
-    ```  
+```csharp
+Properties.Settings.Default.myColor = Color.AliceBlue;  
+```
```
Comment (pkulikov) [inline docs/framework/winforms/advanced/how-to-write-user-settings-at-run-time-with-csharp.md]: With removed leading white spaces the code block would be out of list formatting?

**Comment by BillWagner** [comment]
@mairaw The changes do outdent the code that was part of the lists.

File: docs/framework/winforms/advanced/how-to-read-settings-at-run-time-with-csharp.md [hunk 2]
```diff
@@ -8,18 +8,19 @@ helpviewer_keywords:
 ms.assetid: dbe8bf09-5e1c-49da-9192-154033d7240b
 ---
 # How To: Read Settings at Run Time With C\#
+
 You can read both Application-scoped and User-scoped settings at run time via the Properties object. The Properties object exposes all of the default settings for the project via the Properties.Settings.Default member.  
   
 ## To Read Settings at Run Time with C\#
   
 -   Access the appropriate setting via the Properties.Settings.Default member. The following example shows how to assign a setting named `myColor` to a BackColor property. It requires you to have previously created a Settings file containing a setting named `myColor` of type `System.Drawing.Color`. For information about creating a Settings file, see [How To: Create a New Setting at Design Time](../../../../docs/framework/winforms/advanced/how-to-create-a-new-setting-at-design-time.md).
```
Comment (rpetrusha) [inline docs/framework/winforms/advanced/how-to-read-settings-at-run-time-with-csharp.md]: This is a one-list item, and the code is no longer indented. The bullet can be removed.

File: docs/framework/winforms/advanced/how-to-write-user-settings-at-run-time-with-csharp.md [hunk 3]
```diff
@@ -8,27 +8,27 @@ helpviewer_keywords:
...
+```  
   
-2.  If you want to persist the changes to the settings between application sessions, call the Save method as shown in this example:  
+2. If you want to persist the changes to the settings between application sessions, call the Save method as shown in this example:  
   
-    ```  
-    // C#  
-    Properties.Settings.Default.Save();  
-    ```  
+```csharp
```
Comment (rpetrusha) [inline docs/framework/winforms/advanced/how-to-write-user-settings-at-run-time-with-csharp.md]: This also should be indented.

**Review by rpetrusha — APPROVED** [review]
I've left two comments in addition to the comment from @pkulikov, @mairaw. You can merge once you address them.

File: docs/framework/winforms/advanced/how-to-write-user-settings-at-run-time-with-csharp.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/winforms/advanced/how-to-write-user-settings-at-run-time-with-csharp.md]: Good catch @pkulikov!

**Pull request merged**